README.md

```diff
@@ -62,7 +74,7 @@ author.save(function(err){
 What is module.exports and why do we use it?
 
 ```text
-Your answer...
+It allows yo to reference schema models in other files by requiring the schema.js file
 ```
```

Comment:
:+1: True, but this is not used specifically for Mongoose models. You can export anything out of any `.js` file!